### ISSUE TITLE
 E031 should allow screen-reader content inside glyphicon

### ISSUE THREAD

**Issue by Blackbaud-MatthewBell** [twbs/bootlint#298]
According to E031, an element with the `.glyphicon` class should not contain any content.  However, it seems like it should allow for elements containing content that is targeted only to screen readers using the `.sr-only` class so that they can make sense of the icon element.

As an example:

    <i class="glyphicon glyphicon-star">
        <span class="sr-only>(*)</span>
    </i>

**Comment by zacechola** [comment]
You would want to do something such as the following, rather than nesting the items:

```html
<span class="glyphicon glyphicon-star" aria-hidden="true"></span>
<span class="sr-only">star</span>
```

Here's an example directly pulled from the documentation:

```html
<div class="alert alert-danger" role="alert">
  <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
  <span class="sr-only">Error:</span>
  Enter a valid email address
</div>
```
